src/sentry/models/group.py

```diff
@@ -620,7 +620,7 @@ def is_resolved(self):
         return self.get_status() == GroupStatus.RESOLVED
 
     def has_replays(self):
-        return get_replay_count_for_group(self) > 0
+        return len(get_replay_count_for_group(self)) > 0
```

Comment:
Wait, didn't I make this 'count' fund return an int?

Reply:
fixed this so that the func now returns an int. this code is currently ignored in the type checker. maybe we can fix that.